tests/src/whisk/core/cli/test/SequenceMigrationTests.scala

```diff
@@ -82,11 +82,6 @@ class SequenceMigrationTests
         }
     }
 
-    it should "invoke an old-style sequence (original pipe.js) and get the result" in {
```

Comment:
@ioana-blue since whisk.invoke is now gone - any other pipe.js cleanup we can do?